### PR TITLE
added title to settings page

### PR DESCRIPTION
**Pull request by AkiraLaine**
Can someone check it out and make sure it works? My version of FCC can't run because I can't seem to install a dependency.

### CODE REVIEW

**Comment by alistermada** [comment]
Yep, it works. :+1: 

image

**Comment by AkiraLaine** [comment]
@alistermada Awesome! Thanks.

File: server/boot/user.js [hunk 1]
```diff
@@ -249,7 +249,9 @@ module.exports = function(app) {
   }
 
   function getSettings(req, res) {
-    res.render('account/settings');
+    res.render('account/settings', {
+        title: 'Settings | Free Code Camp'
```
Comment (raisedadead) [inline server/boot/user.js]: Change `title: 'Settings | Free Code Camp'` to `title: 'Settings '`

The tool tip shows an extra `| Free Code Camp` :

Comment (raisedadead) [inline server/boot/user.js]: The changed one will show:

**Comment by raisedadead** [comment]
Closing in favor of #7601

**Pull request closed without merging**